src/pairwise.jl

```diff
@@ -0,0 +1,88 @@
+_allocmatrix(X, Y, T) = Matrix{T}(undef, length(X), length(Y))
+_allocmatrix(X, T) = Matrix{T}(undef, length(X), length(X))
+
+import Distances: pairwise
+
+@doc """
+    pairwise(dist::StringDistance, itr; eltype = Float64, precalc = nothing)
+    pairwise(dist::StringDistance, itr1, itr2; eltype = Float64, precalc = nothing)
```

Comment:
preprocessing instead or precalc? I think verbose is better, especially for options that are not used that often

Reply:
I prefer `preprocess` so changed to that. Ok?

Reply:
yes, thanks